content/id/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -0,0 +1,371 @@
+- apiVersion: "resource.k8s.io/v1beta2"
+  kind: "ResourceSlice"
+---
+
+<!-- overview -->
+
+{{< feature-state feature_gate_name="DynamicResourceAllocation" >}}
+
+Alokasi sumber daya dinamis (_dynamic resource allocation_) adalah API untuk meminta dan berbagi sumber daya antara Pod dan kontainer di dalam sebuah Pod. Ini merupakan generalisasi dari API PersistentVolume untuk sumber daya generik. Biasanya, sumber daya tersebut bisa berupa perangkat seperti GPU.
+
+_Driver_ sumber daya pihak ketiga bertanggung jawab untuk melacak dan mempersiapkan sumber daya, dengan alokasi sumber daya yang ditangani oleh Kubernetes melalui parameter terstruktur (diperkenalkan pada Kubernetes 1.30). Berbagai jenis sumber daya mendukung parameter sembarang untuk mendefinisikan kebutuhan dan inisialisasi.
+
+Kubernetes versi 1.26 hingga 1.31 menyertakan implementasi (alpha) dari DRA klasik, yang sekarang sudah tidak didukung lagi. Dokumentasi ini, yang ditujukan untuk Kubernetes versi {{< skew currentVersion >}}, menjelaskan pendekatan terkini untuk alokasi sumber daya dinamis dalam Kubernetes.
+
+## {{% heading "prerequisites" %}}
+
+Kubernetes versi {{< skew currentVersion >}} menyertakan dukungan API tingkat klaster untuk alokasi sumber daya dinamis, tetapi harus diaktifkan secara eksplisit. Kamu juga harus menginstal _driver_ sumber daya untuk sumber daya tertentu yang akan dikelola menggunakan API ini. Jika kamu tidak menggunakan Kubernetes versi {{< skew currentVersion >}}, periksa dokumentasi untuk versi Kubernetes tersebut.
+
+<!-- body -->
+
+## API
+
+`resource.k8s.io/v1beta1` dan `resource.k8s.io/v1beta2` {{< glossary_tooltip text="API groups" term_id="api-group" >}} menyediakan tipe-tipe berikut:
+
+ResourceClaim
+: Menggambarkan permintaan akses ke sumber daya di dalam klaster untuk digunakan oleh beban kerja (_workload_). Misalnya, jika sebuah beban kerja membutuhkan perangkat akselerator dengan properti tertentu, permintaan tersebut diekspresikan melalui tipe ini. Bagian status melacak apakah klaim ini telah terpenuhi dan sumber daya spesifik apa yang telah dialokasikan.
+
+ResourceClaimTemplate
+: Mendefinisikan spesifikasi dan beberapa metadata untuk membuat ResourceClaim. Dibuat oleh pengguna saat menerapkan beban kerja. ResourceClaim per-Pod kemudian dibuat dan dihapus secara otomatis oleh Kubernetes.
+
+DeviceClass
+: Berisi kriteria seleksi yang telah ditentukan untuk perangkat tertentu serta konfigurasi untuk perangkat tersebut. DeviceClass dibuat oleh administrator klaster saat menginstal _driver_ sumber daya. Setiap permintaan untuk mengalokasikan perangkat dalam ResourceClaim harus merujuk tepat satu DeviceClass.
+
+ResourceSlice
+: Digunakan oleh _driver_ DRA untuk mempublikasikan informasi tentang sumber daya (biasanya perangkat) yang tersedia di klaster.
+
+DeviceTaintRule
+: Digunakan oleh administrator atau komponen control plane untuk menambahkan _taint_ perangkat pada perangkat yang dijelaskan dalam ResourceSlice.
```

Comment:
accoding to @ariscahyadi `control plane` must be written as `bidang kontrol`.

Cmiiw

Reply:
it can be translated to “bidang kontrol” or keep “control plane” with italics.

Reply:
I'll keep "control plane" with italics